### PR TITLE
Add linters

### PR DESCRIPTION
**Pull request by siriwardhanage**
Add linters to Hello-Microverse project

### CODE REVIEW

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @siriwardhanage 👋👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [x] Linters are correctly setup and they are passing👍.
- [x] Github flow have been used👍.
- [x] `index.html` and `styles.css` files have been created👍.
- [x] A `.gitignore` file have been created with the `test.md` and node_modules directory added to it👍.


## Required Changes ♻️

- [ ] Nice job making use of the microverse readme template, but just as it is required, kindly personalise the template so it can serve as a guide to use your project. a well documented readme makes your project easier to use by others and makes it more professional😉. Kindly follow the instructions in the template and create a readme personalised to this project👍.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Bantealem — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️ 

Hi @siriwardhanage ✋🏼 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [x] No LInters error
- [ ] Good commit message

 ## Required Changes ♻️
- [ ] As suggested by the previous reviewer please make sure you customized the `README` template.  
               - Please Remove the Microverse logo it is just for the template.  If you have once you can use it.
Screenshot from 2023-06-02 08-26-32

               - Please add your project name here 
Screenshot from 2023-06-02 08-27-12
              - please remove `server` and `database` from your `readme` and add `HTML` and `CSS` instead of react js
             
Screenshot from 2023-06-02 08-29-29
             - please add key features of your project you have done in this project
             
Screenshot from 2023-06-02 08-29-29
            - Please remove the `live demo` section. Because you don't have the live demo for this project
            
Screenshot from 2023-06-02 08-33-29
     
                    -Please add the necessary information to your project
Screenshot from 2023-06-02 08-35-58
                
            - Please remove the `Deployment` section. Because you don't have a deployment site for this project. Kindly remove it from the head too.
            
Screenshot from 2023-06-02 08-37-05

             - Please replace the link with your actual profiles and also remove `author 2` the 
 from this section.
Screenshot from 2023-06-02 08-39-28
        
                  - Please Add `Futer Feature` to your project
Screenshot from 2023-06-02 08-41-36
                  - Please add your `FAQ` to your readme. If you don't have `FAQ` you can remove it because it is optional.
                  
Screenshot from 2023-06-02 08-42-52
 
         - Please remove unnecessary texts from your README. Remove the below description
         
Screenshot from 2023-06-02 08-45-01

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by siriwardhanage** [comment]
[https://github.com/siriwardhanage/Hello-Microverse#hello-microvere--project](url)

I already update my readme file and commit the change to the main branch you can refer it using above link.Do I have to create new pull request?

File: README.md [hunk 1]
```diff
@@ -7,7 +7,6 @@
 </div>
 
 <!-- TABLE OF CONTENTS -->
-
 # 📗 Table of Contents
```
Comment (Shedrack-Sunday) [inline README.md:11]: - [ ] Good job making the suggested changes suggested by the last reviewer. However, you missed one important section which is `Future_Features`. This section lets you state possible future features the project will need to make it better. Things like sections in the HTML, proper styling, external dependency files, and possibly integration of third party libraries. You're free to put any number but a minimum of three.

- [ ] Changes should be made after the `Author` section, and the part that shows the description should not be added from the image which is present on the template given

## Missing section:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @siriwardhanage    👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done extremely well so far in this project,  However, there's one adjustment you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️  
- Kudus to hitting most of the project requirements ✔️  
  
# Required Changes  🌵 

- [x] You made some impressive implementations which I must say is a job well done. However, there's one I suggest   you make In line with the requirements  then you get approved 👏 

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by siriwardhanage** [comment]
for this project there are no future features required. So I think it is
optional.according to the template instructions I can remove unnecessary
parts. This is my 3rd review.What should I do next?

**Review by victorokoroji — APPROVED** [review]
Hi @siriwardhanage , 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- Requested changes implemented ✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**